tests/licensedcode/data/licenses/gpl-3.0_5.yml

```diff
@@ -0,0 +1,3 @@
+licenses:
+    - gpl-3.0
+template: yes

```

Comment:
Test are not templates. Only rules are.
This is not correct: there should be two licenses: gpl-3.0 plus the other one which is an exception of sorts